ignite/contrib/metrics/precision_recall_curve.py

```diff
@@ -70,6 +59,22 @@ def sigmoid_output_transform(output):
     """
 
     def __init__(self, output_transform: Callable = lambda x: x, check_compute_fn: bool = False) -> None:
+        try:
+            from sklearn.metrics import cohen_kappa_score  # noqa: F401
+        except ImportError:
+            raise RuntimeError("This contrib module requires sklearn to be installed.")
+        self.precision_recall_curve_compute_fn = self.precision_recall_curve_compute()
         super(PrecisionRecallCurve, self).__init__(
-            precision_recall_curve_compute_fn, output_transform=output_transform, check_compute_fn=check_compute_fn
+            self.precision_recall_curve_compute_fn, output_transform=output_transform, check_compute_fn=check_compute_fn
```

Comment:
```suggestion
            self.precision_recall_curve_compute, output_transform=output_transform, check_compute_fn=check_compute_fn
```

why not using directly the function ?